### PR TITLE
PR for review

### PR DESCRIPTION
**Pull request by visva-dev**
## Project Library
- User is able to create books with a title, author, pages parameters.
- Created book is added to the books table and shown in the browser under creation form.
- The user is able to delete the book from the books table.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @visva-dev and @Guy-Gustave 
### You have done well so far :clap: However, you have a few issues to fix:-
- [x] Add linting file with all the required dependencies then add configuration - [follow this guide](https://github.com/microverseinc/linters-config/tree/master/javascript)
- [x] Each Book displayed has a button to toggle the Book read status
- [x] Separate dom manipulation from application logic. Foolw the examples below
### Make the changes and submit for another code review. Good luck.
- Happy coding!

File: script.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  e.preventDefault();
+  const title = document.querySelector('#book-title').value;
+  const author = document.querySelector('#book-author').value;
+  const pages = document.querySelector('#book-pages').value;
+
+  const book = new Book(title, author, pages);
+
+  Book.AddBookToList(book);
+  Book.clearInputs();
+});
```
Comment (rOluochKe) [inline script.js]: ### Separte dom from app logic

**Comment by visva-dev** [comment]
- Fix all issues suggested by Tse.

**Review by lucasmazo23 — CHANGES_REQUESTED** [review]
## Changes required 🔧
### You are doing a great job so far, and the project is looking good.

### For the project to get approved kindly implement these changes:

### Required changes:
- [ ] [REQUIRED] The project still doesn't display the toggle button, you need a button on each book so that after I added it to my library I can change the `read` status

button should be here:
image


image
[source](https://www.theodinproject.com/courses/javascript/lessons/library)

### Optional Changes:
- [ ] [OPTIONAL] In the future, please add the package.json and package-lock.json, so that we ensure we have the same packages installed
image
- [ ] [OPTIONAL] I think you made a better job separating the logic from the DOM manipulation, but I also think there's more room for improvement, separating files and making modules, but since this project doesn't use a packager still, I think this is kind of enough :D
### Happy coding! 💻
## Lucas Mazo

**Review by rOluochKe — APPROVED** [review]
Hello Team
### You got it right, well done :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**